apps/web/src/pages/Pools/PoolAdvertisementPage/components/DeadlineDialog.tsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import InformationCircleIcon from "@heroicons/react/24/solid/InformationCircleIcon";
+
+import { Button, Dialog, Link, LinkProps } from "@gc-digital-talent/ui";
+import { formatDate } from "@gc-digital-talent/date-helpers";
+
+import useRoutes from "~/hooks/useRoutes";
+
+const generateLink = (
+  href: LinkProps["href"],
+  state: LinkProps["state"],
+  chunks: React.ReactNode,
+) => (
+  <Link newTab external href={href} state={state}>
+    {chunks}
+  </Link>
+);
+
+type DeadlineDialogProps = {
+  deadline: Date;
+};
+
+const DeadlineDialog = ({ deadline }: DeadlineDialogProps): JSX.Element => {
+  const intl = useIntl();
+  const routes = useRoutes();
+  // https://stackoverflow.com/a/34602679
+  const localTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <Button
+          mode="inline"
+          color="secondary"
+          icon={InformationCircleIcon}
+          aria-label={intl.formatMessage({
+            defaultMessage: "Learn about how application deadlines work.",
+            id: "8YKsal",
+            description:
+              "Info button label for pool application deadline details.",
+          })}
+        />
+      </Dialog.Trigger>
+      <Dialog.Content>
+        <Dialog.Header>
+          {intl.formatMessage({
+            defaultMessage: "About application deadlines",
+            id: "mQkyM8",
+            description: "Heading for the pool deadlines dialog",
+          })}
+        </Dialog.Header>
+        <Dialog.Body>
+          <div
+            data-h2-display="base(flex)"
+            data-h2-gap="base(x.5)"
+            data-h2-flex-direction="base(column)"
+            data-h2-align-items="base(flex-start)"
+          >
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "In order to provide an equitable opportunity for all Canadians to apply to a position, deadlines close at <strong>11:59 PM Pacific Time</strong> on the date specified.",
```

Comment:
This sentence refers specifically to Canadians: does this mean only Canadians may apply to these processes?

Reply:
Canadian citizens and permanent residents, as far as I'm aware.